Replicator/CookieStore.cc

```diff
@@ -61,11 +84,14 @@ namespace litecore { namespace repl {
                 secure = true;
             } else if (key == "Expires") {
                 if (expires == 0) {
-                    struct tm datetime;
-                    if (strptime(val.c_str(), "%a, %d %b %Y %T %Z", &datetime) == NULL) {
+                    struct tm datetime = { 0 };
```

Comment:
Could you break this whole block out into a function that takes a string and returns a time_t, and put it at the bottom of the file? That would make things cleaner.